qp_klp/tests/test_amplicon_step.py

```diff
@@ -0,0 +1,65 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+from unittest import TestCase
+from qp_klp.AmpliconStep import AmpliconStep
+from sequence_processing_pipeline.Pipeline import Pipeline
+from os.path import join, abspath
+from functools import partial
+from os import makedirs
+import json
+
+
+class AmpliconStepTests(TestCase):
+    def setUp(self):
+        package_root = abspath('./qp_klp')
+        self.path = partial(join, package_root, 'tests', 'data')
```

Comment:
why is this part of the object if it's only use within the setUp? Is the plan to reuse it somewhere else?

Reply:
Good catch! No reason, just an artifact of having been cut and pasted from older code.